test/smoke/src/areas/positron/apps/shiny.test.ts

```diff
@@ -31,7 +31,9 @@ test.describe('Shiny Application', {
 		await app.workbench.quickaccess.openFile(join(app.workspacePathOrFolder, 'workspaces', 'shiny-py-example', 'app.py'));
 		await app.workbench.quickaccess.runCommand('shiny.python.runApp');
 		const headerLocator = app.workbench.positronViewer.getViewerLocator('h1');
-		await expect(headerLocator).toHaveText('Restaurant tipping', { timeout: 20000 });
+		await expect(async () => {
```

Comment:
This is kinda weird? Why not just extend the timeout from 20s to 60s it should accomplish the same thing.

Reply:
No, this is the issue with the frame existence again.  That 20s timeout is not honored if the frame doesn't exist.